src/test/java/core/basesyntax/service/impl/ConverterImpTest.java

```diff
@@ -0,0 +1,75 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.model.ItemTransaction;
+import core.basesyntax.model.Operation;
+import core.basesyntax.service.Converter;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.jupiter.api.Test;
+
+class ConverterImpTest {
+    private final Converter converter = new ConverterImpl();
+
+    @Test
+    void convert_ok() {
```

Comment:
invalid test method naming
```suggestion
    void convert_correctData_ok() {
```